test/integration/targets/azure_rm_containerinstance/tasks/main.yml

```diff
@@ -0,0 +1,101 @@
+- name: Create sample container instance
```

Comment:
Need some more tests with check mode and idempotence runs around removing the instance.